pages/domains/[name]/index.tsx

```diff
@@ -335,26 +212,30 @@ export function ManageNamePageContent({ name }: { name: string }) {
                       <ProfileRecordItem
                         icon={CogSVG}
                         label="manager"
-                        text={ensData?.owner && ensData?.owner}
+                        text={ensData?.owner ?? ""}
                       />
                     </Skeleton>
 
                     <Skeleton>
-                      <ProfileRecordItem
-                        icon={HeartSVG}
-                        label="owner"
-                        text={ensData?.owner}
-                      />
+                      {ensData?.owner && (
+                        <ProfileRecordItem
+                          icon={HeartSVG}
+                          label="owner"
+                          text={ensData?.owner}
+                        />
+                      )}
                     </Skeleton>
 
                     <Skeleton>
-                      <ProfileRecordItem
-                        icon={CalendarSVG}
-                        label="expiry"
-                        text={formatDate({
-                          unixTimestamp: ensData?.expiry / 1000,
-                        })}
-                      />
+                      {ensData?.expiryDate !== undefined && (
+                        <ProfileRecordItem
+                          icon={CalendarSVG}
+                          label="expiry"
+                          text={formatDate({
+                            unixTimestamp: ensData?.expiryDate / 1000,
```

Comment:
Can you create a function with a self-explainable name to do this logic? `ensData?.expiryDate / 1000`